packages/pangea-sdk/PangeaCyber.Tests/ITAuditTest.cs

```diff
@@ -874,4 +874,44 @@ public async Task TestDownloadResults()
         // FIXME: Commented due to Permission denied
         // file.Save("./", file.Filename);
     }
+
+    [Fact]
+    public async Task TestExportDownload()
+    {
+        var exportResponse = await generalClient.Export(new ExportRequest
+        {
+            End = DateTimeOffset.Now,
+            Verbose = false,
+        });
+        Assert.Equal(nameof(ResponseStatus.Accepted), exportResponse.Status);
+
+        // Note that the export can easily take dozens of minutes, if not longer,
```

Comment:
We should limit results to just one day maybe, and try to poll result a `max_retry` taking, maybe, 120 seconds and check if that's enough

Reply:
This worked, thanks.